final_prep/annotations.rb

```diff
@@ -3,11 +3,15 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+# defined a method that includes variables we will use
 def build_a_bear(name, age, fur, clothes, special_power)
+  # variable is equal to the following string interpolation
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # variale is equal to the following array
   demographics = [name, age]
+  # variable is equal to the following string interpolation
   power_saying = "Did you know that I can #{special_power}?"
+#variable set to hash and includes integer, boolean, and strings
```

Comment:
I would still go line by line here. What data type is each value? Where are the values coming from? an argument, or a variable declared above?

Reply:
done - annotated in more detail